SparkPost/2web2ui
# hotfix/FE-1162 - Align items center to fix bug with deployment

Resolves problem with FE-1162 deployment

<img width="170" alt="Screen Shot 2020-08-27 at 12 23 41 PM" src="https://user-images.githubusercontent.com/3613392/91468997-cd793a80-e860-11ea-938d-48dd61db0457.png">


### What Changed
- Vertically center icon within Enable/Disable SSO button

### How To Test
- Go to `/account/settings` and verify the icon and content are vertically centered in both themes
